_posts/2015-08-04-upcoming-events.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Upcoming Events
+date: 2015-08-04 00:20:00
+author: Jessica Grazziotin
+categories: ['event']
+description: There are at least two tech events to check out on Wednesday 8/05! The HNL.io Beta release as well as the Bytemarks weekly radio show on Hawaii NPR featuring a HIcap intern.
```

Comment:
Looks like we're pushing the hnl.io private beta release back another couple of days.